Makefile

```diff
@@ -110,8 +110,11 @@ RBAC_ROOT ?= $(MANIFEST_ROOT)/rbac
 PULL_POLICY ?= Always
 
 # Allow overriding the e2e configurations
-GINKGO_FOCUS  ?=
-GINKGO_NODES  ?= 1
+GINKGO_FOCUS ?=
```

Comment:
can we have a default so when we try to run the e2e test locally, we don't also run the `CAPI` ones?